lk2nd/device/dts/msm8952/msm8937-nokia-nd1.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8937 0x2000>;
+	qcom,board-id = <0x94 0>;
+};
+
+&lk2nd {
+    model = "HMD Global Nokia 5 (nd1)";
+    compatible = "nokia,nd1";
+    lk2nd,match-device = "ND1";
+    lk2nd,match-panel;
+    lk2nd,dtb-files = "msm8937-nokia-nd1";
+    
+
+    gpio-keys {
+    	compatible = "gpio-keys";
+
+        down {
+            lk2nd,code = <KEY_VOLUMEDOWN>;
+            gpios = <&tlmm 91 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+        };
+
+        up {
+			lk2nd,code = <KEY_VOLUMEUP>;
+			gpios = <&pmic_pon GPIO_PMIC_RESIN 0>;
+		};
+
+        power {
+            lk2nd,code = <KEY_POWER>;
+            gpios = <&pmic_pon GPIO_PMIC_PWRKEY 0>;
```

Comment:
Do you need this? should be the default...

Reply:
No, I added it to be certain